examples/operator-manifest.configmap.yaml

```diff
@@ -0,0 +1,141 @@
+kind: ConfigMap
```

Comment:
@tkashem is adding one in #34. I think we can just link to that once #34 is merged.